src/Components/HeaderComp.tsx

```diff
@@ -0,0 +1,22 @@
+// import { useState } from 'react';
+// import { Link } from 'react-router-dom';
+// import { Button } from 'react-bootstrap';
+import logo from '/src/assets/logo.jpeg';
+
+//Landing page header
+//contains image, logo(present on every page & routes back to landing page onClick)
+//
+function LandingHeader() {
+
+    return (
+    <>
+        <div>
+            <body class="bg-50">
```

Comment:
**TODO:** To my understanding, React doesn't need / shouldn't have a <body> tag here -- you can just write it like so:
```
        <div className="bg-50">
            <img src={logo} alt="logo" />
        </div>
```